app/assets/javascripts/static/ContributeDataView.js

```diff
@@ -81,6 +86,63 @@ define([
     cache: function() {
       this.$form = this.$el.find('#new-contribution');
       this.$fieldsets = this.$el.find('.-js-fieldset');
+      this.$fieldSubmit = this.$el.find('#submit');
+      this.$fieldFileUpload = this.$el.find('#fileupload');
+      this.$fieldFileUploaded = this.$el.find('#fileuploaded');
+      this.$fieldFileName = this.$el.find('#fileupload-name');
+    },
+
+    setUploadField: function() {
+      var that = this;
+
+      this.$fieldFileUpload.fileupload({
+        url: '/data/upload',
+        dataType: 'json',
+        autoUpload: true,
+        // acceptFileTypes: /(\.|\/)(gif|jpe?g|png)$/i,
+        maxFileSize: 10000000, // 10 MB
```

Comment:
We should start using constants for things like the max file size, so it's easier to see if its at the top of the view.